variables.tf

```diff
@@ -72,7 +72,7 @@ variable "performance_mode" {
 }
 
 variable "provisioned_throughput_in_mibps" {
-  default     = ""
+  default     = 1.0
```

Comment:
```suggestion
  default     = 0
```

Reply:
i tested it with just `0` and it was not complaining.
@WoodProgrammer  can you please update the default, and then rebuild `README` by executing these commands:

```
make init
make readme/deps
make readme
```

thanks